File: 250624/특정 문자의 등장 횟수/number-appearances-of-a-particular-character.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
    // Please write your code here.
    int cnt1 = 0;
    int cnt2 = 0;
    string s;
    cin >> s;
    for (int i=0; i<s.length()-1; i++) {
        if (s.substr(i,2)=="ee") {
            cnt1 +=1;
        }
        if (s.substr(i,2)=="eb") {
            cnt2 +=1;
        }
    }
    cout << cnt1 << " " << cnt2;
    return 0;
}

File: 250629/특정 문자의 등장 횟수/number-appearances-of-a-particular-character.py
a=input()
cnt1=0
cnt2=0
for i in range(len(a)-1):
    if (a[i]=='e' and a[i+1]=='e'):
        cnt1+=1
    if (a[i]=='e' and a[i+1]=='b'):
        cnt2+=1

print(cnt1,cnt2,sep=" ")